engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -2380,6 +2380,7 @@ namespace dmGraphics
 
         ShaderResourceBinding* res = &module->m_Uniforms[index];
         *type = shaderDataTypeToGraphicsType(res->m_Type);
+        *size = 1; // TODO
```

Comment:
I'm not sure this can be left as a todo, since we'll merge this into the console branches (which use Vulkan), and suddenly the code behaves differently for the users.

Is it hard to implement?